Création biomes désert + création énigme forêt de nuit

Lisouille69 opened the pull request:
Désert : 
- création biomes (oasis, monde abstrait, grotte)
- ajout items dans les biomes 

Forêt de nuit : 
- modification landscape
- création de la "crypte"
- modification des collisions des rochers
- modification position du portail

Grotte : 
- ajout cover dans couloir aller

Lisouille69 committed:
Création des biomes d'énigme du désert

- biome oasis
- biome monde abstrait
- biome grotte

LaReveuseDuFutur committed:
Desert

add item biome monde abstrait, oasis, grotte

Lisouille69 committed:
Création énigme forêt de nuit

- modification landscape
- création de la "crypte"
- modification des collisions des rochers
- modification position du portail
- ajout cover dans couloir aller

HugoPasquier commented:
Pour le désert, le LD est vraiment trop cool, on a vraiment l'impression d'être plongé dans un mini-open world avec une vibe entre God of War Ragnarok et Journey. Le seul truc qui me dérange, c'est la taille de l'émetteur et du "miroir" des lasers. Dans l'idée c'est cool d'avoir des assets de cette taille pour le sentiment de vertige que ça peut procurer. Mais j'ai peur qu'il y ai un effet pétard mouillé quand on croisera les même assets à notre hauteur plus tard dans le jeu. Je pense  qu'on peut juste garder les grandes plateformes, tout en réduisant la taille des éléments laser pour les faire coller à ceux que l'on retrouve dans le Monde Abstrait.

Sinon pour la cover de la grotte, ça ajoute un peu de relief, c'est cool.
Et J'attends de voir ce que vous allez faire avec la crypte de la forêt de Nuit.

adrimyla committed:
Merge branch 'dev_v0.X' into Rework_Level

adrimyla commented:
# Review du LD
## La partie monde abstrait

- Comme Hugo, je trouve que les assets géants ne rendent pas super bien. Pour ceux de la grotte ça passe un peu mieux mais c'est vrai que les émetteus/transmetteurs sont vraiment géants. Je pense qu'il faut garder à l'idée que c'est un avant goût de la suite mais pas "la grande découverte" de ce niveau.
- Pour la position du laser : je comprends pas trop comment on peut le diriger dans le récepteur ? L'emetteur ne vise pas le transmetteur 🤔 
![image](https://github.com/SL-GAME/SansLogiqueGame/assets/57765484/ce1f892a-512e-43f1-b895-63b3f69e4c2a)
- Je suis pas trop pour le faire de poser le robot miroir de façon inclinée, pourquoi ne pas le mettre à plat ou a moitié ensablé ? (surtout que le dessous des pattes est transparent) 
![image](https://github.com/SL-GAME/SansLogiqueGame/assets/57765484/3b566217-47ed-48af-aa33-c78e7e157f96)
Exemple : 
![image](https://github.com/SL-GAME/SansLogiqueGame/assets/57765484/7dc12090-ea48-439a-9626-0cd28cc1dd07)

## La partie grotte
- RAS

## La partie forêt
- Pour la vanne à récupérer il y en a normalement une dans les dossiers que j'avais créé au moment où il fallait en poser une sur la machine. On peut la réutiliser avec son système : Content/Levels/SubLevels/Forest/Blueprints/Machine/BP_ValveKey.uasset
- 
## Autre
- Il faudra ajouter les tuyaux, peut être que @Matda43 peut le faire ? :)
- J'ai pas vraiment compris où sont les différents morceaux de porte ?
- Pour le pilier sous le mannequin, reprendre le celui de la manette : Content/Levels/SubLevels/Grotte/Hall/BP_StoneUnderController.uasset

adrimyla reviewed (changes requested):
# Review du LD

## Rework du désert

### La partie monde abstrait

- Comme Hugo, je trouve que les assets géants ne rendent pas super bien. Pour ceux de la grotte ça passe un peu mieux mais c'est vrai que les émetteus/transmetteurs sont vraiment géants. Je pense qu'il faut garder à l'idée que c'est un avant goût de la suite mais pas "la grande découverte" de ce niveau.
- Pour la position du laser : je comprends pas trop comment on peut le diriger dans le récepteur ? L'emetteur ne vise pas le transmetteur 🤔 
<a href="url"><img src="https://github.com/SL-GAME/SansLogiqueGame/assets/57765484/ce1f892a-512e-43f1-b895-63b3f69e4c2a" align="center" width="350" ></a>
- Je suis pas trop pour le faire de poser le robot miroir de façon inclinée, pourquoi ne pas le mettre à plat ou a moitié ensablé ? (surtout que le dessous des pattes est transparent) 
<a href="url"><img src="https://github.com/SL-GAME/SansLogiqueGame/assets/57765484/3b566217-47ed-48af-aa33-c78e7e157f96" align="center" width="350" ></a>
Exemple : 
<a href="url"><img src="https://github.com/SL-GAME/SansLogiqueGame/assets/57765484/7dc12090-ea48-439a-9626-0cd28cc1dd07" align="center" width="350" ></a>

###  La partie grotte
- RAS
- La cover dans la grotte est nickel 👍

### La partie forêt
- Pour la vanne à récupérer il y en a normalement une dans les dossiers que j'avais créé au moment où il fallait en poser une sur la machine. On peut la réutiliser avec son système : Content/Levels/SubLevels/Forest/Blueprints/Machine/BP_ValveKey.uasset

### Autre
- Il faudra ajouter les tuyaux, peut être que @Matda43 peut le faire ? :)
- J'ai pas vraiment compris où sont les différents morceaux de porte ?
- Pour le pilier sous le mannequin, reprendre le celui de la manette : Content/Levels/SubLevels/Grotte/Hall/BP_StoneUnderController.uasset

## Rework forêt de nuit

- Pour ce qui est de la forêt de nuit on en reparle lundi

Lisouille69 committed:
Modifs suite aux retours PR

- Réduction taille assets Monde Abstrait
- Rotation robot miroir
- Changement BP Valve
- Changement BP Stone sous le mannequin

Lisouille69 committed:
resize assets + laser atelier monde abstrait ok

adrimyla committed:
Merge branch 'dev_v0.X' into Rework_Level

adrimyla committed:
Placing dummy accessible for the moment

adrimyla committed:
Relocating portal barrier in night forest

adrimyla committed:
Merge branch 'dev_v0.X' into Rework_Level

# Conflicts:
#	Content/Levels/SubLevels/Desert/DesertTemp.umap
#	Content/Levels/SubLevels/Forest/ForestByNight.umap
